app/explainer_server/explainer.py

```diff
@@ -0,0 +1,43 @@
+import os
+import time
+import subprocess
+
+# File path
+CURRENT_DIR = os.getcwd()
+UPLOAD_FOLDER = os.path.abspath(os.path.join(CURRENT_DIR,'app', 'uploads'))
```

Comment:
I think I saw this in another file..
sounds to me like you can have a "const" file to save all of this constants